ports/stm32/Makefile

```diff
@@ -336,12 +336,18 @@ SRC_O += \
 	resethandler_m0.o \
 	shared/runtime/gchelper_m0.o
 else
+ifeq ($(MCU_SERIES),g0)
```

Comment:
Instead of duplicating code, this could be rolled in to the `f0` block above (using `filter`).